packages/osfamilymap.yaml

```diff
@@ -30,8 +30,7 @@ RedHat:
         - epel
       pkgs:
         - gcc
-        - python-devel
-        - python2-pip
+        - python36-pip
```

Comment:
@noelmcloughlin as we're [doing it now in the images builder](https://github.com/netmanagers/salt-image-builder/blob/master/.travis.yml#L39), it seems that the package name is to use is `python3-pip`, mind you change it?

Reply:
That's done @javierbertoli  
Nice work with https://github.com/netmanagers/salt-image-build - I'll have a look sometime.